README.md

```diff
@@ -44,10 +44,10 @@ For more details, see [Ledger's documentation](https://support.ledger.com/hc/en-
 
 # Installing Ledger App from Release
 
-You can download `.hex` files from the
+Please download `.hex` files from the
```

Comment:
The user might be confused by this, the thing they are downloading is 'nano-s-release.tar.gz'

Reply:
Another fix for you!